README.md

```diff
@@ -513,25 +513,10 @@ Oh, you betcha. Here's a partial list:
 
 ## Does `ko` work with [OpenShift Internal Registry](https://docs.openshift.com/container-platform/latest/registry/registry-options.html#registry-integrated-openshift-registry_registry-options)?
```

Comment:
I think once we get this and the other any-k8s-internal-registry publishers in, we can just move these up to the [Naming Images](https://github.com/google/ko#naming-images) section above, where `ko.local` and `kind.local` are documented.

Reply:
Yeah, I guess at some point we should just list the specialcases we integrate with.